src/app/components/forms/dynamic-form/dynamic-switch/terra-dynamic-switch.component.ts

```diff
@@ -22,6 +22,8 @@ export class TerraDynamicSwitchComponent
     @Input() public inputFormFields:Array<TerraFormFieldBase<any>>;
     @Input() public inputFormGroup:FormGroup;
     @Input() public inputSubSwitch:boolean;
+    @Input() public inputPortletHeader:string;
+    @Input() public inputUsePortlet:boolean;
```

Comment:
Folgendes: Der Standardfall der Dynamic Form wird sein, eine eigene Ansicht mit der generierten Form bereitzustellen (nicht wie im Content-Builder in einem Filter). Du solltest aber für einen Standardfall nicht immer einen input setzen müssen, um auch das entsprechende (Standard-)Aussehen zu erreichen. Sprich: Bitte die Variable in `inputWithoutPortlet` umbenennen und standardmäßig die Form immer in ein Portlet packen.

Reply:
es ist standardmäßig auf true, warum sollte ich da was ändern?

Reply:
siehe terra-dynamic-form.component.ts